pkg/security/resolvers/tags/resolver.go

```diff
@@ -96,7 +96,9 @@ func NewResolver(config *config.Config) Resolver {
 			log.Errorf("unable to configure the remote tagger: %s", err)
 		} else {
 			return &DefaultResolver{
-				tagger: remote.NewTagger(options),
+				// TODO: components. We should use the telemetry component instead of nil
```

Comment:
I had a quick look and it doesn't seem too long to inject the telemetry instance in this function.
What do you think about fixing that in a followup PR?
Happy to help.

Reply:
Absolutely. It was part of follow up work for this component 😄